doc/HOWTO-KAFKA.md

```diff
@@ -81,7 +81,13 @@ Then from the Sonar root directory, after building it, run Sonar in daemon mode
 config file in the Sonar shell:
 
 ```
-  target/debug/sonar daemon <config-file>
+  target/debug/sonar daemon tests/kafka/sonar-slurm-node.ini
```

Comment:
These changes can and should be moved to the previous patch, since the ini files are available there.